pkg/utils/etcdutil/etcdutil.go

```diff
@@ -865,6 +865,16 @@ func (lw *LoopWatcher) load(ctx context.Context) (nextRevision int64, err error)
 	if limit != 0 {
 		limit++
 	}
+	if err := lw.preEventsFn([]*clientv3.Event{}); err != nil {
```

Comment:
It seems that there is no implemented function using the param.

Reply:
It is used to judge whether is called by `watch` or by `load`.